docs/source/hardware/lowpriority.rst

```diff
@@ -0,0 +1,79 @@
+
+.. _partition_names:
+
+**********
+Low priority partition
+**********
+:sub:`Updated September, 2022`
+
+
+Introduction
+===================
+``lowpriority`` is a new partition on Discovery that allows the research community to use private resources on the HPC cluster when they are idle. 
+This new partition has hardware not otherwise available to the general research community and, in time, could double the resources available to NURC users. 
+This is a common practice in HPC clusters to optimize the use of idle private resources that consume power and cooling. 
+
+When to use the low priority partition
+===================
+
+From most to least-recommended scenarios:
+
+1. Code that can be checkpointed
```

Comment:
and maybe add the corresponding RC links here as well? unless they are already mentioned elsewhere here.